src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -163,51 +165,61 @@ export const CheckoutPayment = ({
   const [isLoading, setIsLoading] = useState(false)
   const [isError, setIsError] = useState(false)
 
-  const onConnectPaymentSuccess = useCallback(
+  const addPaymentToCart = useCallback(
     async (paymentTokenId) => {
       try {
         await addPaymentTokenMutation({
           variables: {
             id: quoteCartId,
             paymentTokenId,
           },
+          refetchQueries: ['QuoteCart'],
         })
       } catch (error) {
         console.error('Failed to add Payment Token :', error.message, error)
       }
+    },
+    [addPaymentTokenMutation, quoteCartId],
+  )
 
-      try {
-        setIsLoading(true)
-        const { data } = await startCheckout({
-          variables: { quoteIds, quoteCartId },
-        })
-        if (data?.quoteCart_startCheckout.__typename === 'BasicError') {
-          console.error('Could not start checkout')
-          setIsError(true)
-        }
-        // Poll for Status
-        getStatus({
-          variables: {
-            quoteCartId,
-          },
-        })
-      } catch (error) {
-        const isManualReviewRequired = checkIsManualReviewRequired(
-          (error.graphQLErrors || []) as GraphQLError[],
-        )
-        if (isManualReviewRequired) {
-          throw new Error('Manual Review required')
-        }
-        setIsLoading(false)
+  const checkout = useCallback(async () => {
```

Comment:
I think this variable name is too generic -- can we be more descriptive?

Reply:
I meant `checkout` as a verb as in "do checkout", since this is calling `startCheckout`. Maybe this is unclear since it can be a noun and a verb. Do you have a suggestion?

Reply:
Hmmm... maybe just `handleCheckout` or `performCheckout`